palace/utils/geodata.cpp

```diff
@@ -212,7 +285,7 @@ void RefineMesh(const IoData &iodata, std::vector<std::unique_ptr<mfem::ParMesh>
   // reorient only the coarse mesh so that the refinements are still true refinements of
   // the original mesh (required for geometric multigrid). Otherwise, it happens after
   // refinement.
-  if (iodata.model.reorient_tet && mesh.capacity() > 1)
+  if (iodata.model.reorient_tet_mesh && mesh.capacity() > 1)
```

Comment:
The check at https://github.com/awslabs/palace/pull/260/files#diff-0642b48eb210cdd7e98a30345d4857eb87c192985610bc00696294583cd09a43R259 should prevent that from ever being the case. The idea here is that if we are storing a multigrid hierarchy of meshes, we want to reorient the coarse mesh, otherwise we will wait until the end to do the reorientation.